node-definitions/openbsd/README.md

```diff
@@ -0,0 +1,245 @@
+# OpenBSD-CML
+Instructions for building an OpenBSD node in Cisco Modeling Lab
+
+## Why OpenBSD in CML?
+
+OpenBSD is standards-compliant, has good documentation, and is secure.
+
+OpenBSD can act as a router, a routed layer 3 firewall, a NAT boundary, a
+bridging Layer-3 firewall, an end host, a webserver, a load balancer, and many
+other roles not mentioned here.
+
+# Create a qcow2 OpenBSD VM
+
+## Create the VM
+As resources in CML are generally scarce I suggest starting with a fairly
+minimal configuration for a VM:
+
+* RAM: 1GB
+* Disk: 8GB
+* CPU: Single
+* Network: single interface with Internet access
```

Comment:
One of the use cases you spell out above is router.  Your node def file does provide for multiple interfaces.  Your default if count is 2.  Why state a single interface here?

Reply:
I've updated the 'Create the VM' section to address this.